src/fiddle.ts

```diff
@@ -95,16 +96,43 @@ export class FiddleFactory {
     return new Fiddle(path.join(folder, 'main.js'), 'entries');
   }
 
-  public async create(src: FiddleSource): Promise<Fiddle | undefined> {
-    if (src instanceof Fiddle) return src;
+  public async create(
+    src: FiddleSource,
+    options?: { packAsAsar?: boolean },
```

Comment:
Existing style in this codebase would dictate that these options should be defined on a separate interface rather than inline.